package.json

```diff
@@ -74,7 +74,7 @@
   },
   "peerDependencies": {
     "react": ">=0.14",
-    "react-with-direction": "^1.1.0"
+    "react-with-direction": "^1.3.0"
```

Comment:
technically this is a breaking change; we should lower the runtime version to 1.1 instead.